### DISCUSSION TITLE
AudioGridder retro compatibility?

### DISCUSSION THREAD

**Discussion by abelbascu** [apohl79/audiogridder#626] [Q&A]
I haven't found the question anywhere. If it exists, please close this thread.

My concern is:

What if I use xxx.xx version of AG now in my project and let's say I keep updating Audiogridder and once we are in 2023 I attempt to open this very same project that used an old version of AG? would AG still work?

Thanks in advance for your insights,
Abel

**Comment by apohl79** [comment]
Yes. AG stays backwards compatible.
